jinja2/filters.py

```diff
@@ -914,6 +914,14 @@ def do_rejectattr(*args, **kwargs):
     """
     return _select_or_reject(args, kwargs, lambda x: not x, True)
 
+def do_json(value, **kwargs):
+    """Serializes an object as JSON. Optionally given keyword arguments
+    are passed to json.dumps(), ensure_ascii however defaults to False.
+    """
+    import json
```

Comment:
Not a big fan of this. Now you have a `json` filter that always uses the stdlib `json` module with the default settings and absolutely no way (except defining your own filter) to override this.